.travis.yml

```diff
@@ -5,6 +5,11 @@ rvm:
   - '2.1.8'
   - '2.2.4'
   - '2.3.0'
+  - '2.4.1'
+
+before_install:
```

Comment:
this is nitpicking a little but, but may want to change this from `before_install` to `install`.  i think installing stuff like `rubocop` is what the `install` event hook is for.  heres the docs for the [travis config file](https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Installation-Step)